.github/workflows/dynamodb.yml

```diff
@@ -47,7 +48,7 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
     - name: Build
       run: |
-        cargo build --locked --features aws
+        cargo build --locked --features aws,test
```

Comment:
What does the `test` feature flag have to do with the optimization level?

Reply:
`cargo test` is going to inject the `test` feature at the next step.